src/components/ExperienceTimeline.module.css

```diff
@@ -0,0 +1,51 @@
+.timeline {
```

Comment:
Do we need this explicit CSS file? I think we can provide these styles in component files directly using Chakra UI's props. We can extract style components separately if adding this much style bloats the main ExperienceTimeline component